pkgs/by-name/cu/cups-brother-dcpt720dw/package.nix

```
@@ -0,0 +1,123 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  perl,
+  ghostscript,
+  coreutils,
+  gnugrep,
+  which,
+  file,
+  gnused,
+  dpkg,
+  makeWrapper,
+  libredirect,
+  debugLvl ? "0",
+}:
+
+let
+  printerName = "DCP-T720DW";
+  model = "dcpt720dw";
+  version = "3.5.0-1";
+  debHash = "sha256-ToUFGnHxd6rnLdfhdDGzhvsgFJukEAVzlm79hmkSV3E=";
+in
+# Based on cups-brother-dcpt725dw package
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
you already took all repeating variables outside of the mkDerivation call